allauth/socialaccount/providers/quickbooks/provider.py

```diff
@@ -0,0 +1,60 @@
+from allauth.socialaccount import app_settings
```

Comment:
Can you add this provider to `test_settings.py`, `overview.rst` (list of supported providers) and `installation.rst` ?